src/controllers/import.js

```diff
@@ -156,6 +222,10 @@ function ImportController(context) {
         urls, options, importScript, customHeaders,
       } = multipartFormData;
 
+      validateUrlsByDomain(urls, profile.getScopes());
+
+      log.debug(`Creating a new import job with ${urls.length} URLs.`);
```

Comment:
Is there any reason this is a debug-level log and not info?

Reply:
+1 to that Shikha.  Having the logs tell us the story in Coralogix is critical.

Reply:
Could modify it to an info log. I kept it as debug, to not pollute the logs.

Reply:
Logs are critical to the tracing story.  It's critical that we are able to track the progress of the job.  It's absolutely okay to be logging information like this, and I would encourage more of it so that we can clearly see what's happening.